.github/workflows/update-badges.yml

```diff
@@ -38,14 +38,14 @@ jobs:
       run: |
         mkdir -p ../.img
         wget https://img.shields.io/badge/total_tests-$(pytest --collect-only -q | tail -1 | cut  -d ' ' -f 1)-green?style=flat-square -O ../badges/total-count.svg
-        wget https://img.shields.io/badge/interop_tests-$(pytest --collect-only -m "interop_uc_sanity and client_connect" -q | tail -1 | cut  -d '/' -f 1)-green?style=flat-square -O ../badges/interop-count.svg
-        wget https://img.shields.io/badge/sanity_tests-$(pytest --collect-only -m "uc_sanity and not interop" -q | tail -1 | cut  -d '/' -f 1)-green?style=flat-square -O ../badges/sanity-count.svg
+        wget https://img.shields.io/badge/interop_tests-$(pytest --collect-only -m "ow_sanity_interop" -q | tail -1 | cut  -d '/' -f 1)-green?style=flat-square -O ../badges/interop-count.svg
+        wget https://img.shields.io/badge/sanity_tests-$(pytest --collect-only -m "ow_sanity_lf" -q | tail -1 | cut  -d '/' -f 1)-green?style=flat-square -O ../badges/sanity-count.svg
         wget https://img.shields.io/badge/performance_tests-$(pytest --collect-only -m "performance" -q | tail -1 | cut  -d '/' -f 1)-green?style=flat-square -O ../badges/performance-count.svg
         wget https://img.shields.io/badge/advance_tests-$(pytest --collect-only -m "advance" -q | tail -1 | cut  -d '/' -f 1)-green?style=flat-square -O ../badges/advance-count.svg
-        wget https://img.shields.io/badge/regression_tests-$(pytest --collect-only -m "regression and not interop" -q | tail -1 | cut  -d '/' -f 1)-green?style=flat-square -O ../badges/regression-count.svg
+        wget https://img.shields.io/badge/regression_tests-$(pytest --collect-only -m "ow_regression_lf" -q | tail -1 | cut  -d '/' -f 1)-green?style=flat-square -O ../badges/regression-count.svg
 #        wget https://img.shields.io/badge/interop_performance_tests-$(pytest --collect-only -m "interop_uc_sanity and client_connectivity" -q | tail -1 | cut  -d '/' -f 1)-green?style=flat-square -O ../badges/performance-interop-count.svg
         
-    - name: commit changes
+    - name: commit changes"
```

Comment:
Double quotes will probably break the workflow, please delete it

Reply:
@dunaev-opsfleet Those Double quotes were there since from the previous commits and workflow is running fine with them. Please take a look into this https://github.com/Telecominfraproject/wlan-testing/actions/runs/3739997778 and confirm the action.